apps/fluent-tester/src/TestComponents/Shadow/ShadowDepthTestSection.tsx

```diff
@@ -10,12 +10,12 @@ import { useFluentTheme } from '@fluentui-react-native/framework';
 const getThemedStyles = themedStyleSheet(() => {
   return {
     effectBox: {
-      width: 366,
+      maxWidth: 732,
```

Comment:
where does this number come from?  it workes fine for smallest screen like iphonese?

Reply:
Adding this property doesn't visibly impact small screens, I added it so that the test boxes wouldn't get overly wide on big screens - without it the width of the test boxes will keep increasing with the width of the screen. It's an arbitrary number I just did 366*2, 366 comes from [Figma](https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=4553%3A38) (although the design of this test page was never intended to follow that Figma)

Reply:
Description was a little misleading, just updated it